### PR TITLE
New package: atom-bin-1.32.2

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/atom-bin/files/LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2011-2018 GitHub Inc.
```
Comment (Vaelatern) [inline srcpkgs/atom-bin/files/LICENSE]: No. We must NOT track licenses in our repo.

File: srcpkgs/atom-bin/files/atom.desktop [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```
Comment (Vaelatern) [inline srcpkgs/atom-bin/files/atom.desktop]: Or .desktop files.

File: srcpkgs/atom-bin/template [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Template file for 'atom-bin'
+pkgname=atom-bin
```
Comment (Vaelatern) [inline srcpkgs/atom-bin/template]: What's wrong with the atom package?

File: srcpkgs/atom-bin/template [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+depends="git gtk+ GConf libnotify libXtst nss python xdg-utils
+ desktop-file-utils alsa-lib libgnome-keyring libXScrnSaver"
+short_desc="Hackable text editor for the 21st century"
+conflicts="atom"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="MIT"
+homepage="https://atom.io/"
+only_for_archs="x86_64"
+nostrip=yes
+nocross=yes
```
Comment (Vaelatern) [inline srcpkgs/atom-bin/template]: Also I think `nonfree` since we didn't do the build.

File: srcpkgs/atom-bin/REMOVE [hunk 5]
```diff
@@ -0,0 +1,9 @@
+if [ "$UPDATE" = "no" ]; then
+    case "$ACTION" in
+        post)
+            if [ -d usr/share/atom ]; then
+                rm -rf usr/share/atom
+            fi
+        ;;
+    esac
+fi
```
Comment (Vaelatern) [inline srcpkgs/atom-bin/REMOVE]: Why this?

**Comment by shizonic** [comment]
sorry guys but I just wanted to share it because iwork for me. but if packages like this aren't welcome please let me know. so to be concrete: any chance that this pkg ever get merged if i fix your suggestions?

File: srcpkgs/atom-bin/files/LICENSE (hunk 1 above)
Comment (shizonic) [reply srcpkgs/atom-bin/files/LICENSE]: And what if the archive doesn't contain one?

File: srcpkgs/atom-bin/files/atom.desktop (hunk 2 above)
Comment (shizonic) [reply srcpkgs/atom-bin/files/atom.desktop]: And what if the archive doesn't contain one?

File: srcpkgs/atom-bin/template (hunk 3 above)
Comment (shizonic) [reply srcpkgs/atom-bin/template]: Nothing wrong just not up to date.

File: srcpkgs/atom-bin/template (hunk 4 above)
Comment (shizonic) [reply srcpkgs/atom-bin/template]: So this means?

File: srcpkgs/atom-bin/REMOVE (hunk 5 above)
Comment (shizonic) [reply srcpkgs/atom-bin/REMOVE]: Because the directory does get removed otherwise.

File: srcpkgs/atom-bin/template (hunk 3 above)
Comment (Vaelatern) [reply srcpkgs/atom-bin/template]: Then please try to update that one instead?

File: srcpkgs/atom-bin/template (hunk 4 above)
Comment (Vaelatern) [reply srcpkgs/atom-bin/template]: I guess not `nonfree`, so no change.

File: srcpkgs/atom-bin/REMOVE (hunk 5 above)
Comment (Vaelatern) [reply srcpkgs/atom-bin/REMOVE]: This is a bug: https://github.com/void-linux/xbps/issues/23
and don't have the REMOVE file just to fix this. xbps will be fixed to handle it.

Comment (shizonic) [reply srcpkgs/atom-bin/REMOVE]: Nice to know. Thanks.

**Comment by maxice8** [comment]
@Vaelatern ok ?

File: srcpkgs/atom-bin/template (hunk 3 above)
Comment (Vaelatern) [reply srcpkgs/atom-bin/template]: This was not resolved, but I can let it slide....

**Comment by Vaelatern** [comment]
I hate it, but OK.

File: srcpkgs/atom-bin/template (hunk 3 above)
Comment (maxice8) [reply srcpkgs/atom-bin/template]: i'd rather not let it slide

Comment (Vaelatern) [reply srcpkgs/atom-bin/template]: In that case: I think this PR can be closed, because the correct course is for us to build atom.

**Pull request closed without merging**